### DISCUSSION TITLE
"Preview Code" is not working

### DISCUSSION THREAD

**Discussion by bennycode** [damien-schneider/cuicui#23] [General]
I was checking the [Droppable File](https://cuicui.day/common-ui/inputs) example today and clicked on "Preview Code". It is showing this to me:

> An error occurred while reading file from path /var/task/src/ui/cuicui/common-ui/inputs/droppable-file/variant1.tsx:
{"errno":-2,"code":"ENOENT","syscall":"open","path":"/var/task/src/ui/cuicui/common-ui/inputs/droppable-file/variant1.tsx"}

**Comment by ionutcnu** [comment]
+1

**Comment by damien-schneider** [comment]
Thank you for reporting this ! Everything is fixed, and a test have been written to check this not happen anymore 😉
